tab_downloads.md

```diff
@@ -9,11 +9,11 @@ tags: wstg
 
 ## Stable
 
-View the always-current stable version at [/stable](/stable).
+View the always-current stable version at [/stable](stable/).
 
-## [Unreleased 4.1] - 2020
+## [Version 4.1] - 2020-04-01
```

Comment:
Needs to be updated to contain the PDF when the build job runs when we tag, and date update.

Reply:
Yup yup!